src/kbmod/search/pydocs/stamp_creator_docs.h

```diff
@@ -120,6 +120,14 @@ namespace pydocs {
       The co-added stamp.
   )doc";
 
+  static const auto DOC_StampCreator_get_coadded_stamps = R"doc(
+  todo
+  )doc";
+
+  static const auto DOC_StampCreator_filter_stamp = R"doc(
+  todo
+  )doc";
+
 } /* pydocs */
```

Comment:
Optional: Up to you, but it's probably nice to fill these out while we're here

Reply:
Good call. I added them.